src/redux/features/actions/updateProfile.js

```diff
@@ -0,0 +1,24 @@
+import instance from "../../axiosinstance";
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+const token =  localStorage.getItem("token");
+// const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7Im5hbWUiOiJBZG1pbiBUcm9qYW4iLCJlbWFpbCI6ImFkbWluMTIzQGdtYWlsLmNvbSIsImlkIjoxLCJyb2xlIjoiYWRtaW4iLCJzdGF0dXMiOiJhY3RpdmUiLCJwcm9maWxlUGljIjoiaHR0cHM6Ly9yZXMuY2xvdWRpbmFyeS5jb20vZG1qeHVreDA5L2ltYWdlL3VwbG9hZC92MTY3NTg0NDY5Mi9wcm9maWxlcy9Qcm9maWxlLUF2YXRhci1QTkctRnJlZS1Eb3dubG9hZF9wYXFmcmYucG5nIiwibGFzdFRpbWVQYXNzd29yZFVwZGF0ZWQiOiIyMDIzLTAzLTE3VDE0OjAzOjE0LjQwMFoifSwiaWF0IjoxNjgwODUwOTIwfQ.2URytQCe6jh5NtDBXWqRdzdr7IFFu87c5_uPrZbGPR4";
```

Comment:
can you remove this comment since it's not being used

Reply:
solved my friend @Ntarecedrick